examples/sticky.ads.amp.html

```diff
@@ -160,7 +159,7 @@
     </div>
   </header>
   <main role="main">
-  <amp-sticky-ad layout="nodisplay">
+  <amp-sticky-ad layout="container">
```

Comment:
The current code is updated. Here's what I found, after setting `display:flex` through css classList, I still find `style = "display: none"` on the HTML element.